Update direct-routing-plan.md (#2738)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan.md

Microsoft is working on adding additional certification authorities based on customer requests. 

## SIP Signaling: FQDNs 

> Direct Routing is offered in the following Office 365 envirionments:

Reviewer: ```suggestion
Direct Routing is offered in the following Office 365 environments:
```

> - Office 365;

Reviewer: ```suggestion
- Office 365
```

> - Office 365 GCC;

Reviewer: ```suggestion
- Office 365 GCC
```

> - Office 365 GCC High;

Reviewer: ```suggestion
- Office 365 GCC High
```

- Office 365 DoD

> More about GCC, GCC High and DoD envirionments [on this link] (https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government)

Reviewer: ```suggestion
Find out more about GCC, GCC High and DoD environments [on this link] (https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government).
```

> More about GCC, GCC High and DoD envirionments [on this link] (https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government)

Reviewer 2: ```suggestion
Learn more about [Office 365 and US Government environments](https://docs.microsoft.com/office365/servicedescriptions/office-365-platform-service-description/office-365-us-government/office-365-us-government) such as GCC, GCC High, and DoD.
```

> ### Office 365 and Office 365 GCC envirionments

Reviewer: ```suggestion
### Office 365 and Office 365 GCC environments
```

- 52.114.7.24 
- 52.114.14.70

> You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN sip-all.pstnhub.microsoft.com resolves to all the IP addresses above. 

Reviewer: ```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN sip-all.pstnhub.microsoft.com resolves to all these IP addresses. 
```

> ### Office 365 GCC DoD envirionment

Reviewer: ```suggestion
### Office 365 GCC DoD environment
```

The connection point for Direct Routing is the following FQDN:

> **sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. Note as the Office 365 DoD envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs

Reviewer: ```suggestion
**sip.pstnhub.dod.teams.microsoft.us** – Global FQDN. As the Office 365 DoD environment exists only in the US data centers, there is no secondary and tertiary FQDNs.
```

The FQDNs – sip.pstnhub.dod.teams.microsoft.us will be resolved to one of the following IP addresses:

- 52.127.64.33
- 52.127.68.34

> You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all the IP addresses above. 

Reviewer: ```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.dod.teams.microsoft.us resolves to all these IP addresses. 
```

> ### Office 365 GCC High envirionment

Reviewer: ```suggestion
### Office 365 GCC High environment
```

The connection point for Direct Routing is the following FQDN:

> **sip.pstnhub.gov.teams.microsoft.us** – Global FQDN. Note as the GCC High envirionment exists only in the US datacenters, there is no secondary and tertiary FQDNs

Reviewer: ```suggestion
**sip.pstnhub.gov.teams.microsoft.us** – Global FQDN. As the GCC High environment exists only in the US data centers, there is no secondary and tertiary FQDNs.
```

The FQDNs – sip.pstnhub.gov.teams.microsoft.us will be resolved to one of the following IP addresses:

- 52.127.88.59
- 52.127.92.64

> You will need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all the IP addresses above. 

Reviewer: ```suggestion
You need to open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.  If your firewall supports DNS names, the FQDN  sip.pstnhub.gov.teams.microsoft.us resolves to all these IP addresses. 
```

## SIP Signaling: Ports

> Prt requiremtns are the same for all Office 365 envirionments, where Direct Routing is offered:

Reviewer: ```suggestion
Port requirements are the same for all Office 365 environments where Direct Routing is offered:
```

> - Office 365;

Reviewer: ```suggestion
- Office 365
```

> - Office 365 GCC;

Reviewer: ```suggestion
- Office 365 GCC
```

> - Office 365 GCC High;

Reviewer: ```suggestion
- Office 365 GCC High
```

## Media traffic: Port ranges

> Note the requirements below apply if you ant to deploy Direct Routing without Media Bypass. For firewall requirements for Media Bypass please refer to [Plan for Media Bypass with Direct Routing](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-plan-media-bypass)

Reviewer: ```suggestion
Note that the requirements below apply if you want to deploy Direct Routing without Media Bypass. For firewall requirements for Media Bypass, please refer to [Plan for media bypass with Direct Routing](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-plan-media-bypass).
```

The media traffic flows to and from a separate service in the Microsoft Cloud. The IP range for Media traffic:

> ### Office 365 and Office 365 GCC envirionments

Reviewer: ```suggestion
### Office 365 and Office 365 GCC environments
```

- 52.112.0.0 /14 (IP addresses from 52.112.0.1 to 52.115.255.254).

> ### Office 365 GCC DoD envirionment

Reviewer: ```suggestion
### Office 365 GCC DoD environment
```

- 52.127.64.0/21

> ### Office 365 GCC High envirionment

Reviewer: ```suggestion
### Office 365 GCC High environment
```

- 52.127.88.0/21

> ### Port range (applicable to all envirionments)

Reviewer: ```suggestion
### Port range (applicable to all environments)
```
